src/components/List/Cells/ReviewerActionCell.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { CSSProperties } from 'react'
+import { Link } from 'react-router-dom'
+import { Label } from 'semantic-ui-react'
+import { CellProps } from '../../../utils/types'
+import strings from '../../../utils/constants'
+
+const ReviewerActionCell: React.FC<CellProps> = ({
+  application: {
+    reviewAssignedNotStartedCount,
+    reviewAssignedCount,
+    reviewDraftCount,
+    reviewPendingCount,
+    reviewSubmittedCount,
+    reviewAvailableForSelfAssignmentCount,
+    serial,
+    isFullyAssignedLevel1,
+  },
+}) => {
+  const actions = []
+  const not0 = (num: number | undefined) => num && Number(num) !== 0
```

Comment:
Personal preference: `notZero`, but up to you.

Reply:
Can change this now, not problem.

Reply:
Done: 7312770